scripts/zones/PsoXja/mobs/Diremite.lua

```diff
@@ -0,0 +1,12 @@
+-----------------------------------
+-- Area: Pso'Xja
+--  Mob: Diremite
+-----------------------------------
+local ID = require("scripts/zones/PsoXja/IDs");
+
+function onMobDeath(mob, player, isKiller)
+end;
+
+function onMobDespawn(mob)
+    tpz.mob.phOnDespawn(mob, ID.mob.GYRE_CARLIN_PH, 5, math.random(3600*4, 3600*5)) -- 4-5 hours.
```

Comment:
Where did you get 4-5 hours? I'm only seeing as low as 15-30 mins on wikia, and again I don't think lottery mobs should have an upper bound on their spawn time.

Reply:
Oh sorry, I thought I pasted the clean version not the Dawnbreak version.